### DISCUSSION TITLE
flutter ios build issue

### DISCUSSION THREAD

**Discussion by NikhilIfanow** [codemagic-ci-cd/codemagic-docs#2417] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

check my yaml file

### Expected results

ios build publish on appstore in testflight

### Actual results

error in ios build failed

### Build id (optional)

65f7f9ad939bb78f64154c35

**Comment by himesh-cm** [comment]
It's being discussed on discord [here](https://discord.com/channels/1131597315707261018/1217572457754398852).

**Reply by NikhilIfanow** [reply]
okay
